styles.css

```diff
@@ -5,3 +5,27 @@
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --logo-background: #0a0a23;
+}
+
+nav {
+  position: absolute;
+  width: 100%;
+  height: 80px;
```

Comment:
I think we can lessen the height here so it matches more closely with the actual forum page navbar. 50px would be better